base/server/python/pki/server/cli/user.py

```diff
@@ -353,6 +354,81 @@ def execute(self, argv):
             del_see_also=del_see_also)
 
 
+class UserRemoveCLI(pki.cli.CLI):
+
+    def __init__(self, parent):
+        super(UserRemoveCLI, self).__init__(
+            'del',
+            'Remove %s user' % parent.parent.name.upper())
+
+        self.parent = parent
+
+    def print_help(self):
+        print('Usage: pki-server %s-user-del [OPTIONS] <user ID>' % self.parent.parent.name)
+        print()
+        print('  -i, --instance <instance ID>       Instance ID (default: pki-tomcat).')
+        print('  -v, --verbose                      Run in verbose mode.')
+        print('      --debug                        Run in debug mode.')
+        print('      --help                         Show help message.')
+        print()
```

Comment:
I created PR #3807 to try the f-string, it looks nice but too bad we can't use it as a docstring. Also, if we use `argparse` the help message will be autogenerated. For now I've moved the help message into a `help` class variable (so that it's easier to read) and use `format()` to substitute the subsystem name. What do you think?